sdk/core/performance-stress/CMakeLists.txt

```diff
@@ -2,27 +2,37 @@
 # SPDX-License-Identifier: MIT
 
 cmake_minimum_required (VERSION 3.13)
-project(azure-perf-stress LANGUAGES CXX)
+project(azure-performance-stress LANGUAGES CXX)
```

Comment:
Why is it "performance-stress" and not "perf-framework" or "performance-framework".

Reply:
the name comes from the .Net version. We can change it, I think. I don´t think there are guidelines for it